xlators/cluster/dht/src/dht-common.h

```diff
@@ -50,6 +50,18 @@
 /* Rebalance nodeuuid flags */
 #define REBAL_NODEUUID_MINE 0x01
 
+#define RESET_LOCK_ARRAY(_array)                                               \
```

Comment:
I think a `static inline` function would be better in this case. Macros have the problem that its arguments may be evaluated more than once, which could introduce performance issues or even unwanted side effects if the caller doesn't take care (which is hard to enforce).

Reply:
Done